### DISCUSSION TITLE
gsc/ExternalTools or gsc/WidgetMask in Windows 10

### DISCUSSION THREAD

**Discussion by Enlight432** [goldendict/goldendict#1691] [Q&A]
What is the difference between the following two Grabers in OCR in the GoldenDict++ program?
gsc/ExternalTools
gsc/WidgetMask
Which ones are better for Windows 10?

**Comment by nonwill** [comment]
WidgetMask is **NOT** supported on **macOS**. 
![mv](https://github.com/goldendict/goldendict/assets/46510529/1e1ad989-82d5-44ab-bd10-6bdf7c2206b9)
**ExternalTools** which captures screenshot and then sends the image to clipboard, can be any **exe** or **cmd/shell script** configured by user:
`wingraber` is the default exe on Windows, `screencapture` is the default on macOS, and `scrot/xclip` on Linux.

For **Windows 10** or Linux: 
WidgetMask is better or more efficient than the default ExternalTool(wingraber) which is just a wraper of WidgetMask.

If WidgetMask and the default ExternalTool (wingraber) are both not working, any screen capture tools existed on Windows can replace them.

**Comment by vedgy** [comment]
Discuss GoldenDict++ elsewhere. Also reconsider using the illegal proprietary fork: #1424.
